app/controllers/v1/ministries_controller.rb

```diff
@@ -9,6 +9,15 @@ def index
       render_ministries
     end
 
+    def update
```

Comment:
What is preventing non-superadmin from running this?

Need tests for that, too.

Reply:
Fail. Will have to refactor authorization for this. Either fix the power so that it's not ministry specific and can be utilized in the context of all controllers...or just use the class method `User.superadmin?` circumventing power use entirely and further decentralizing authorization in the app. The latter seems awful from a design standpoint...but the former will require more heavy refactoring. Will touch base tomorrow.